styleguide/source/_patterns/03-organisms/four-up-teaser.twig

```diff
@@ -1,6 +1,6 @@
 <div class="ama__four-up-teaser">
-  {% for item in fourUpTeaser.items[0:4] %}
-    {% set articleStub = item %}
+  {% for article in fourUpTeaser %}
+    {% set articleStub = article %}
```

Comment:
These items may not always be articles on the D8 website. It is probably best to just keep it as item.